index.ts

```diff
@@ -1,20 +1,28 @@
 //original version from https://github.com/evanw/esbuild/blob/plugins/docs/plugin-examples.md
-import { preprocess, compile, VERSION } from "svelte/compiler";
+import { preprocess, compile, VERSION, compileModule } from "svelte/compiler";
 import { dirname, basename, relative } from "path";
 import { promisify } from "util";
 import { readFile, statSync } from "fs";
 import { originalPositionFor, TraceMap } from "@jridgewell/trace-mapping";
 
-import type { CompileOptions, Warning } from "svelte/types/compiler/interfaces";
-import type { PreprocessorGroup } from "svelte/types/compiler/preprocess";
+import type { CompileOptions, ModuleCompileOptions, CompileResult } from "svelte/compiler";
+import type { PreprocessorGroup } from "svelte/compiler";
 import type { OnLoadResult, Plugin, PluginBuild, Location, PartialMessage } from "esbuild";
 
+type Warning = CompileResult["warnings"][number];
+
 interface esbuildSvelteOptions {
     /**
      * Svelte compiler options
      */
     compilerOptions?: CompileOptions;
 
+    /**
+     * Svelte compiler options for module files (*.svelte.js and *.svelte.ts)
+     */
+
```

Comment:
Nit: extra line here so it isn't getting picked up in the `.d.ts` file

Reply:
Fixed!